Dockerfile.test

```diff
@@ -0,0 +1,60 @@
+FROM registry.scc.suse.de/suse/sles15:ga
+
+RUN useradd --no-log-init --create-home scc
+
+ARG REGCODE
+RUN SUSEConnect -r $REGCODE
+RUN zypper --non-interactive ar http://download.opensuse.org/repositories/openSUSE:/Tools/SLE_15/openSUSE:Tools.repo
+RUN zypper --non-interactive --gpg-auto-import-keys ref
+RUN zypper --non-interactive up
+RUN zypper --non-interactive in -t pattern devel_osc_build
+RUN zypper --non-interactive install --no-recommend wget curl timezone \
+    gcc-c++ libffi-devel git-core zlib-devel libxml2-devel libxslt-devel libmariadb-devel \
+    mariadb-client ruby2.5-rubygem-bundler make build sudo ruby-devel
+
+
+ENV DOCKERIZE_VERSION v0.6.0
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+ARG OBS_USER
+ARG OBS_PASSWORD
```

Comment:
Might be cleaner to put all `ARG`s at the top, so they are easier to see.
(This is not a full review ;-) )

Reply:
fixed